repositories/account.go

```diff
@@ -1,3 +1,4 @@
+//go:generate mockgen -source=account.go -destination=../services/test/mock/mock_account.go -package=mock
```

Comment:
I think we should use one directory for mocks.
Either /test/mock or just /mock (like in the next file)